bin/create_fortran_test.py

```diff
@@ -0,0 +1,149 @@
+"""
+
+See docs/TEST.md for usage of this script
+
+Use this script to create and initial <exercise>_test.f90 file
+which can be used as a skeleton for your test. Typically, you will
+have to replace 'response' in the generated file with the right
+function call.
+
+Also note that fortran has issues with special characters such as \n and \t
+so take special care handling these.
+
+
+Prerequsites
+- Working cmake and fortran compiler
+- Python3.x (it may work with Python2 but I have not tested)
+- latest version of https://github.com/exercism/problem-specifications.git
+
+Work flow for creating a new test
+- pull latest changes from exercism/problem-specifications
+- run this script for the example you want to create
+- copy config/CMakeLists.txt for exercise directory
+- implement working exercise
+- fix potential problematic tests (see eg. exercise/bob "Test 20" and "Test 24")
+- ensure ctest validates without errors
+- open a pull request with your changes
+
+For bob example:
+
+$ python3 bin/create_fortran_test.py -j ../../../exercism/problem-specifications/exercises/bob/canonical-data.json -t exercises/bob/bob_test.f90
+Namespace(json='../../../exercism/problem-specifications/exercises/bob/canonical-data.json', target='exercises/bob/bob_test.f90')
+Wrote : exercises/bob/bob_test.f90
+$ cp config/CMakeLists.txt exercises/bob/.
+$ cd exercises/bob
+$ touch bob.f90
+$ mkdir Debug
+$ cd Debug
+$ cmake ..
+$ make
+$ ctest -V
+"""
+
+import json
+import argparse
+
+def fix_and_quote_fortran_multiline(txt):
+    """Fortran cant handle multiple, so adding continuation character '&'
```

Comment:
```suggestion
    """Fortran can't handle multiple, so adding continuation character '&'
```

But it can't handle multiple what?